lib/Basics/ReadWriteSpinLock.h

```diff
@@ -29,6 +29,7 @@
 #include "Basics/SharedAtomic.h"
 #include "Basics/SharedCounter.h"
```

Comment:
I think `SharedAtomic` and `SharedCounter` are not needed here...?!

Reply:
Yes, they were used in the old `ReadWriteSpinLock` implementation, but this is no longer the case.